ui/pages/reservation-unit/[...params].tsx

```diff
@@ -302,140 +322,89 @@ const ReservationUnitReservation = ({
     setPendingReservation,
   ]);
 
-  const [
-    deleteReservation,
-    { data: deleteData, loading: deleteLoading, error: deleteError },
-  ] = useMutation<
+  const [deleteReservation] = useMutation<
     { deleteReservation: ReservationDeleteMutationPayload },
     { input: ReservationDeleteMutationInput }
   >(DELETE_RESERVATION, {
     errorPolicy: "all",
+    onCompleted: () => {
+      setDataContext(null);
+      setPendingReservation(null);
+      router.push(`${reservationUnitPrefix}/${reservationUnit.pk}`);
+    },
+    onError: () => {
+      setDataContext(null);
+      setPendingReservation(null);
+      router.push(`${reservationUnitPrefix}/${reservationUnit.pk}`);
```

Comment:
Is there a reason the error is not displayed or handled differently to success?

Reply:
The rationale here was that the reservation may or may not exist (it will be deleted by the backend after n minutes) and we didn't want to expose the error of removing a non-existent reservation to the user.